content/global/inclusive.mdx

```diff
@@ -0,0 +1,111 @@
+
+These fundamental tools are useful when designing new features, as well as auditing existing ones.
+
+### Scope out minimal viable safety tools
+
+Account for how the feature you’re designing could be used to harm others, and how to mitigate harm in stress cases.
+- [ ] Is there an escape hatch? How easy is it to stop the stress from spreading?
+- [ ] Is there an easy way to report abuse and block the offending user(s)?
+
+### Add trust signals
+
+How might we remind people of the humans behind the code?
+- Usage of avatars where appropriate
+- Usage of full names, not just handles, in workplace environments
+- Mutual collaborators
+- Mutual followers/following
+- Shared languages, contribution, repos, orgs
+- Account age (I know this one is tricky)
+- Profile completeness (verified email, uploaded profile photo)
+
+### Privacy
+
+- [ ] When capturing personal identifying information (PII), make it clear _why_ we need the information and _how_ it will be used. This can be done as a sentence or two of copy with a link to help docs
+- [ ] Clearly describe when PII will be public or private
+- [ ] Clear ability to opt in and out of features and notifications
+
+### Clear, accessible, and inclusive language
+
+Be mindful of people who are not fluent in English, or read less-than-fluently. It's especially important to keep language as clear as possible during **high-stress moments**. Also, when introducing technical jargon, be sure to provide documentation wherever possible.
+
+In addition to accessible language, you can rely on existing frameworks to ensure inclusive language is being used in feature work. Whenever you allow expressiveness with a freeform text box, consider:
+- [ ] Blocking words and symbols on the ADL hate symbols list: https://www.adl.org/hate-symbols
+- [ ] Using existing blocked words. Example: Blocked topics: https://data.githubapp.com/sql/7142000a-07e0-457a-9da3-a88c8ebf94e2
+
+### Include documentation and onboarding
+
+> Documentation is highly valued, frequently overlooked, and a means for establishing inclusive and accessible communities. ([Open Source Survey](https://opensourcesurvey.org/2017/#insights))
+
+Closely related to the above section on inclusive language, we have a community that is also in various stages of their programming or open source contribution journey. Well written documentation and onboarding helps GitHub users feel welcome and productive, and is a great addition to any new feature.
+
+For example, the detailed [CLI Manual](https://cli.github.com/manual/) is a wonderful, inclusive companion to the CLI tool that expands beyond the GitHub help documentation.
+
+## Preparing to ship
+
+When working on feature landing pages or social media posts, provide diverse stories and feature developers that are representative of the community.
+- [ ] Who are you including? Who are you leaving out?
+- [ ] Would someone from a marginalized background see this content and feel represented?
+
+## After shipping
+
+🚧
```

Comment:
Could we have a final section with a very short list of hand picked resources to read more about this topic, with a tiny blurb for why we picked each? For example the Design for Real Life book from A List Apart came to mind, or Technically Wrong.